app/config.go

```diff
@@ -9,6 +9,12 @@ import (
 	"github.com/odpf/salt/config"
 )
```

Comment:
@rahmatrhd for config.yaml file, should we put it in the root project under the name config.yaml.example ?

Reply:
don't remember putting config.yaml here, I think it is used for documentation, and also because it is a config for "app", another config we have is for "cmd", maybe to segregate those CMIIW